Dockerfile

```diff
@@ -8,9 +8,11 @@ RUN useradd --shell /bin/sh --user-group weblate \
   && mkdir -p /home/weblate/.ssh \
   && touch /home/weblate/.ssh/authorized_keys \
   && chown -R weblate:weblate /home/weblate \
-  && chmod 700 /home/weblate/.ssh \
-  && install -d -o weblate -g weblate -m 755 /usr/local/lib/python3.7/dist-packages/data-test \
-  && install -d -o weblate -g weblate -m 755 /app/data
+  && chmod 700 /home/weblate/.ssh
+
+# I dunno what this line is doing, please document this.
+RUN install -d -o weblate -g weblate -m 755 /usr/local/lib/python3.7/dist-packages/data-test \
```

Comment:
AFAIR this is needed to run the testsuite in the container with stock test settings.